code/game/objects/effects/spawners/abnormality_room.dm

```diff
@@ -32,6 +32,7 @@
 		ACM.c_tag = "Containment zone: [abno_datum.name]"
 	SSabnormality_queue.postspawn()
 	SSlobotomy_corp.NewAbnormality(abno_datum)
+	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_ABNORMALITY_SPAWN, abno_datum)
```

Comment:
Ideally, it should be moved to the lobotomy_corp subsystem NewAbnormality proc.

Reply:
I moved it to the NewAbnormality proc after the abno is added to the global list